### ISSUE TITLE
Setup process requires creation of admin user - NOT Working

### ISSUE THREAD

**Issue by wege2003** [antedebaas/DMARC-Reports#89]
Hi everyone,

in the setup process the admin user is asked to create, but this does not work in the current version, you are always redirected to the error page after the creation. But the user isnt in the database also. Can you fix this or how can I access the application logs?

Screenshot:
![image](https://github.com/antedebaas/DMARC-SMTPTLS-Reports/assets/31622452/4b6d781b-7981-448d-802b-83c3d73ae3c7)

Screenshot 2:
![image](https://github.com/antedebaas/DMARC-SMTPTLS-Reports/assets/31622452/8b135772-940a-4e18-8bfa-004e91d22d03)

Thanks, 
Greets Michael

**Comment by wege2003** [comment]
It seems to be something with the database, because when i create manually the user it shows up in the db with the correct id.
Screenshot:
![image](https://github.com/antedebaas/DMARC-SMTPTLS-Reports/assets/31622452/4f3de4f6-f752-4233-9b53-1a1b8e5030c9)


But in the setup process it can create my user:

```
2024-05-08 19:26:08.751 UTC [441] ERROR:  null value in column "id" of relation "users" violates not-null constraint
2024-05-08 19:26:08.751 UTC [441] DETAIL:  Failing row contains (null, __redacte-email__, ["ROLE_ADMIN"], __redacred-pw-hash__, t).
```

Please fix that.

Thanks!

**Comment by antedebaas** [comment]
I’ll look into this, what database type are you using?

**Comment by wege2003** [comment]
i am using postresql, but it happens with mysql also

**Comment by antedebaas** [comment]
i could not replicate it on mysql but the postgres databse was indeed wrong.
this should be fixed in 1.12.2
let me know if you still encounter problems.

**Comment by wege2003** [comment]
now it is working fine, DMARC reports are loading, and the mailbox check is also working. The user creation works also fine.

Thanks, for the fix!
